apps/communications/dialer/style/buttons.css

```diff
@@ -72,10 +72,6 @@ html[dir="rtl"] li .activity {
   text-align: right;
 }
 
-html[dir="rtl"] li .activity.icon {
```

Comment:
I remove this part because the padding cause incorrect layout under RTL and I have no idea about the orignal porpose of padding. Hi @DouglasSherk, can you remember why we need this padding here?